android/assets/jsons/translations/template.properties

```diff
@@ -1187,6 +1187,16 @@ Number of your cities celebrating\n'We Love The King Day' thanks\nto access to t
 WLTK demand = 
 WLTK- = 
 Number of your cities\ndemanding this resource for\n'We Love The King Day' = 
+Social Policies: = 
+At war with [enemy] = 
+Friends with [civName] = 
+[numberOfTurns] Turns Left = 
+Denounced [otherCiv] = 
+Allied with [civName] = 
+Civilization Info = 
+Social Policies = 
+Wonders = 
```

Comment:
`Wonders` is already an entry in this file, as is `Social policies` (note the lowercase P)

Reply:
fixed :)